install/firefly-install.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: kristocopani
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+    curl \
+    mc \
+    sudo
+curl -sSLo /usr/share/keyrings/deb.sury.org-php.gpg https://packages.sury.org/php/apt.gpg
+echo "deb [signed-by=/usr/share/keyrings/deb.sury.org-php.gpg] https://packages.sury.org/php/ bookworm main" >/etc/apt/sources.list.d/php.list
+$STD apt-get update
+$STD apt-get install -y \
+    apache2 \
+    php8.3 \
+    php8.3-cli \
+    libapache2-mod-php8.3 \
+    php8.3-{bcmath,intl,curl,zip,gd,xml,mbstring,mysql} \
+    mariadb-server \
+    composer
```

Comment:
Split into Default dependencies (Apache2, madiadb above) 

And then create a new MSG Section "Setup PHP" with only the PHP stuff